### ISSUE TITLE
[Question] Export to NVM via CLI

### ISSUE THREAD

**Issue by Ntweat** [colmap/colmap#252]
Hi
I wanted to ask how to export the model to NVM via command line. I could not find the documentation anywhere.
I tried 
> image_undistorter --output_type NVM

But that is not an option.

**Comment by Ntweat** [comment]
Figured it out. It was under model_converter

**Comment by ahojnnes** [comment]
Thanks for the feedback, let me know if there are other questions.

**Comment by Enderych** [comment]
Hello, in fact, I still have similar questions. After automatic reconstruction, now I have a sparse document which includes cameras.bin, images.bin and points3D.bin. I can use model_converter to transfer them to '.txt', and then read the information. But I can not convert them into 'NVM' format.  The error shows 'Check failed: file.is_open() path_to_output'.
Then I tried to use colmap GUI, firstly input model from fused.ply, then output model with 'NVM' model, but then I found that there are only points information inside the nvm document, no cameras and images poses information. How can I get an nvm with both cameras and points?
